src/Resources/config/doctrine/Wishlist.orm.xml

```diff
@@ -11,7 +11,7 @@
         </id>
         <field name="name" type="string" column="name" nullable="true" />
 
-        <field name="token" type="string" column="token" unique="true" />
+        <field name="token" type="string" column="token" />
```

Comment:
Why token is no longer unique?

Reply:
If user is anonymous (not logged in) - token determines if wishlist is assigned to him. Back in time we had single wishlist it was acceptable to make token unique - now it is impossible.

Reply:
Token should be always unique!  This is an API identifier